trash.go

```diff
@@ -367,6 +367,10 @@ func chanPackagesFromLines(lnc <-chan string) <-chan util.Packages {
 func listPackages(rootPackage string) util.Packages {
 	r := util.Packages{}
 	filepath.Walk(rootPackage, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			logrus.Error(err)
+			return err
+		}
```

Comment:
@karlmutch Can you move this change into a separate PR? Good catch. I'll merge immediately.